index.html

```diff
@@ -139,60 +44,152 @@ <h2 class="about-me-h2"> About <br> Myself </h2>
+            <img class="client-section-counter" src="ressources/Counter.png" alt="Counter">
+            <li class="project-year"id="info-client">2015</li>
+          </ul>
+          <p class="project-info">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul class="language-used">
+            <li><div class="tags">html</div></li>
+            <li><div class="tags">css</div></li>
+            <li><div class="tags">javaScript</div></li>
+          </ul>
+        <button type="button" class="button">See Project!</button>
       </section>
     </section>
-    <footer>
-    </footer>
+
+<!-- About Me Section  -->
+    <section class="about-me">
+      <h2 class="about-me-h2"> About <br> Myself </h2>
+      <p class="about-me-p"> Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitiate to contact me.</p>
+      <div class="social">
+        <div class="let-connect">LET’S CONNECT</div>
+        <ul class="icon-list">
+          <li><a href="https://facebook.com/benj.p009" target="_blank" rel="noopener" class="fa fa-facebook"></a></li>
+          <li><a href="https://instagram.com/benjamin_patin" target="_blank" rel="noopener" class="fa fa-instagram"></a></li>
+          <li><a href="https://linkedin.com/in/benjaminpatin" target="_blank" rel="noopener" class="fa fa-linkedin"></a></li>
+          <li><a href="https://twitter.com/benjp009" target="_blank" rel="noopener" class="fa fa-twitter"></a></li>
+          <li><a href="https://github.com/benjp009" target="_blank" rel="noopener" class="fa fa-github"></a></li>
+        </ul>
+      </div>
+      <button type="button" class="button" id="button">Get Resume</button>
+      <div class="self-category">
+        <button class="knowledges">Languages</button>
+          <ul class="dropdown">
+            <li>
+              <ul class="knowledges-details">
+                <img class="knowledge-img" src="ressources/Javascript.png">
+                <p class="knowledge-p"> javaScript<p>
+              </ul>
+              <ul class="knowledges-details">
+                <img class="knowledge-img" src="ressources/HTML.png">
+                <p class="knowledge-p"> HTML<p>
+              </ul>
+              <ul class="knowledges-details">
+                <img class="knowledge-img" src="ressources/CSS.png">
+                <p class="knowledge-p"> CSS<p>
+              </ul>
+            </li>
+          </ul>
+        <h3 class="knowledges">Frameworks</h3>
+          <ul class="dropdown">
+            <li></li>
+          </ul>
+        <h3 class="knowledges">Skills</h3>
+          <ul class="dropdown">
+            <li></li>
+          </ul>
+      </div>
+    </section>
+
+<!-- Contact Me Section  -->
+    <section class="contact-me-white">
```

Comment:
The contact section paragraph is narrowed in the center and the form is not in the middle. Background images should be in the same position as in the design.
![contact-y](https://user-images.githubusercontent.com/35504149/139495397-78efb3fc-e547-432c-903c-e302dac013e4.png)

Reply:
in 1440px the background image is at the same position...